mujina-idp/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jdk-alpine
+ENV SERVICE_HOST localhost
+
+VOLUME /tmp
```

Comment:
I'm not clear on the purpose of mounting a temp volume?

Reply:
I believe  Ican remove it, however if you want some background, please search for "Containerize It" on this page: https://spring.io/guides/gs/spring-boot-docker/

_We added a VOLUME pointing to "/tmp" because that is where a Spring Boot application creates working directories for Tomcat by default. The effect is to create a temporary file on your host under "/var/lib/docker" and link it to the container under "/tmp". This step is optional for the simple app that we wrote here, but can be necessary for other Spring Boot applications if they need to actually write in the filesystem._

Reply:
Ok thanks, lets leave it in then in case we add changes in the future which need to write there